Sampled/UI/VIews/CombinedPlayerView.swift

```diff
@@ -0,0 +1,260 @@
+//
+//  CombinedPlayerView.swift
+//  Sampled
+//
+//  Created by GitHub Copilot on 11/13/25.
+//
+
+import SwiftUI
+
+struct CombinedPlayerView: View {
+  let track: LibraryInfoTrackModel
+  let isPlaying: Bool
+  let onPlayTapped: () -> Void
+  let onPrevious: () -> Void
+  let onNext: () -> Void
+  @State private var showFullInfo = false
+  @FocusState private var isFocused: Bool
+  
+  var body: some View {
+    VStack(spacing: 0) {
+      // Player bar
+      HStack(spacing: 16) {
+        // Album artwork
+        let image = switch track.albumArtwork {
+          case .empty, .mixed: NSImage()
+          case let .value(artwork): artwork?.image ?? NSImage()
+        }
+        
+        Button {
+          withAnimation(.snappy(duration: 0.25)) {
+            showFullInfo.toggle()
+          }
+        } label: {
+          Image(nsImage: image)
+            .resizable()
+            .aspectRatio(contentMode: .fill)
+            .frame(width: 64, height: 64)
+            .clipShape(RoundedRectangle(cornerRadius: 6))
+            .shadow(color: .black.opacity(0.2), radius: 4, y: 2)
+            .overlay(alignment: .bottomTrailing) {
+              Image(systemName: showFullInfo ? "chevron.down.circle.fill" : "info.circle.fill")
+                .font(.caption)
+                .padding(4)
+                .background(.ultraThinMaterial, in: Circle())
+                .padding(2)
+            }
+        }
+        .buttonStyle(.plain)
+        .help("Show/hide track details")
+        
+        // Track info
+        VStack(alignment: .leading, spacing: 4) {
+          switch track.title {
+            case .empty:
+              Text("No track selected")
+                .font(.headline)
+                .foregroundStyle(.secondary)
+            case .value(let title):
+              Text(title ?? "Unknown")
+                .font(.headline)
+                .lineLimit(1)
+            case .mixed:
+              Text("Multiple tracks")
+                .font(.headline)
+                .foregroundStyle(.secondary)
+          }
+          
+          HStack(spacing: 4) {
+            switch track.artistName {
+              case .empty:
+                EmptyView()
+              case .value(let artist):
+                Text(artist ?? "Unknown Artist")
+                  .font(.subheadline)
+                  .foregroundStyle(.secondary)
+                  .lineLimit(1)
+              case .mixed:
+                Text("Various Artists")
+                  .font(.subheadline)
+                  .foregroundStyle(.tertiary)
+                  .lineLimit(1)
+            }
+            
+            if case .value(let albumName) = track.albumName, let name = albumName {
+              Text("•")
```

Comment:
I believe most players use an em (or en?) dash to separate artist name and track name. Can you do a similar thing here? If possible, I would prefer this be localized, which can be done with string interpolation. The reason is that HStack doesn't use the same logic behind spacing as text does. I have code in the project for this, so I can show you how.